roles/connectivity_troubleshooter_igw/README.md

```diff
@@ -1,22 +1,26 @@
-Role Name
+connectivity_troubleshooter_igw
 =========
 
 A role to troubleshoot connectivity issues between AWS resources in an Amazon VPC and an internet resource using an internet gateway.
 
+* Confirms whether the source has a public IP address associated with the destination resource.
+* Confirms whether the security group rules allow the needed traffic from the source to the destination resource.
+* Confirms whether the network ACLs allow the needed traffic from the source resource.
```

Comment:
Not really sure if we should list these info here. The module eval_src_gateway_route module does that. I will probably suggest we generate the documentation for the module.

Reply:
Per [the Jira ticket](https://issues.redhat.com/browse/ACA-1081), the point of this PR is to clarify what it is these roles are doing. These steps are ultimately what the role is doing (via the module) to identify potential sources of connectivity issues. I think the purpose of adding these here is that someone using the role shouldn't have to go into the module to get a general overview of how the role is troubleshooting connectivity.

Reply:
I would rather prefer keep this description here, the end user is supposed to know the internal design of the role so having the validation steps here in the documentation is helpful